README.md

```diff
@@ -83,15 +83,16 @@ ones controlled by your test.
 
 We provide a top-level method called `td.replace()` that operates in two
 different modes: CommonJS module replacement and object-property replacement.
-Both modes will, by default, perform a deep clone the real dependency which
-replaces all functions it encounters with fake test double functions that can be
-configured by your test to either stub responses or assert invocations.
+Both modes will, by default, perform a deep clone of the real dependency,
+replacing all functions it encounters with fake test double functions. These
+fake test double functions can be configured by your test to either stub
+responses or assert invocations.
 
 #### Module replacement with Node.js
 
 **`td.replace('../path/to/module'[, customReplacement])`**
 
-If you're using Node.js and don't mind using the CommonJS `require()` method in
+If you're using Node.js and don't mind using the CommonJS `require()` function in
```

Comment:
`require` is _technically_ a method, I suppose, but "function" seems more normal from a JS standpoint. Also, we call it a function in another place:
>The test suite (usually in a global after-each hook) must call `td.reset()` to ensure the real `require()` function and dependency modules are restored after each test case.